Formula/mariadb-connector-odbc.rb

```diff
@@ -0,0 +1,24 @@
+class MariadbConnectorOdbc < Formula
+  desc "MariaDB Connector/ODBC is a standardized, LGPL licensed database driver using the industry standard ODBC API."
```

Comment:
description should not end in a full stop (audit) and this is too long (audit)

Reply:
Oh whoops, forgot to re-audit!

Reply:
The `MariaDB Connector/ODBC is ` is an audit failure in spirit in that we have an audit to prevent "$formula_name is " which won't kick in due to the space and slash and capitalization but which we should still follow